### PR TITLE
Frilled Lizard

### PR DESCRIPTION
**Pull request by jacksondefa**
I rewrote description of Frilled Lizard to make it better suited to tile

### CODE REVIEW

File: crawl-ref/source/dat/descript/monsters.txt [hunk 1]
```diff
@@ -1121,8 +1121,7 @@ proudly, despite its lingering fear and unkempt appearance.
 %%%%
 frilled lizard
 
-A lizard with a giant and colourful fringe folded around its neck. It flares
-its impressive frill at its victims while hissing angrily.
+A sleek lizard with darting eyes from whose neck snaps tight a fan of rattaling gold and red scales. It hisses angrily and whips its tail.
```
Comment (aegolden) [inline crawl-ref/source/dat/descript/monsters.txt]: Small typo here:  "rattaling" should be "rattling"

**Comment by isloat** [comment]
FYI frilled lizards are using the old giant newt tile and are slated for a new tile.

**Comment by jacksondefa** [comment]
Thanks for noticing!!

**Comment by jacksondefa** [comment]
My mate john is at the moment designing a new tile for it. Saw a glimpse of it yesterday on his computer monitor. :)

**Comment by isloat** [comment]
It's been nearly two months and there's been no updates on this, so I'm closing it. Reopen it if that tile ever gets finished.

**Pull request closed without merging**